HIP-Basic/runtime_compilation/README.md

```diff
@@ -27,33 +29,39 @@ The diagram below summarizes the runtime compilation part of the example.
 17. The first few elements of the result vector $y$ are printed to the standard output.
 
 ![hiprtc.svg](hiprtc.svg)
```

Comment:
Unrelated to the change, but the `[...]`-part should contain the image description.